README.md

```diff
@@ -13,6 +13,12 @@ The service is configured with environment variables, see [config.py](config.py)
 
 Development defaults can be used by setting `ENVIRONMENT=DEV`.
 
+When running the application in your IDE, you may get a log warnings similar to:
```

Comment:
This wasn't introduced here, this happens when you run it in Pycharm. I've just updated the README to have some information about why it's happening.